e2e/test/Helpers/ConsoleEventListener.cs

```diff
@@ -1,89 +1,50 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
 
 namespace System.Diagnostics.Tracing
 {
     public sealed class ConsoleEventListener : EventListener
     {
-        private readonly string[] _eventFilters;
+        private static readonly string[] _eventFilters = new string[] { "DotNetty-Default", "Microsoft-Azure-Devices", "Azure-Core", "Azure-Identity" };
```

Comment:
Azure-Core and identity will be required later when we merge feature to here. But no harm keeping it in the list. We won't even get those events so should be a no-op.